tests/test_deprecations.py

```diff
@@ -167,3 +167,9 @@ def test__check_time_unit():
             # remove xscale from pyfar 0.6.0!
             create_figure()
             pf.plot._utils._check_time_unit(None)
+
```

Comment:
Can you also add a test to check if the function was deleted if the pyfar version if 0.8.0 or larger. I think we did that for the other cases. Here's an example:
```
if version.parse(pf.__version__) >= version.parse('0.5.0'):
        with pytest.raises(AttributeError):
            # remove get_nearest_k() from pyfar 0.5.0!
            coords.get_nearest_k(1, 0, 0)
```